Elasticity.C

```diff
@@ -1218,11 +1218,11 @@ std::string ElasticityNorm::getName (size_t i, size_t j,
       j -= 5;
       char comp[16];
       if (!anasol)
-        sprintf(comp,"a(u^h,w%zu)",j);
+        sprintf(comp,"a(u^h,w%u)",uint8_t(j));
```

Comment:
Wouldn't it be better/easier then to just increase the buffer size instead of introducing the uint8_t conversion?
Np point being that stingy on the bytes here.

Reply:
we can do that instead, sure.